packages/bridge-ui-v2/config/sample/configuredBridges.example

```diff
@@ -8,7 +8,7 @@
         "erc20VaultAddress": "",
         "erc721VaultAddress": "",
         "erc1155VaultAddress": "",
-        "crossChainSyncAddress": "",
+        "taikoAddress": "",
```

Comment:
Not sure this rename was necessary, the relayer also refers to those as CrossChainSync and the contracts implement the `ICrossChainSync` interface.

Reply:
Reverted the change.